### PR TITLE
Update php lint workflows

### PR DESCRIPTION
**Pull request by artonge**
Php linting workflow is currently failing: https://github.com/nextcloud/activity/runs/4824472725?check_suite_focus=true

This PR updates php linting workflows as per https://github.com/nextcloud/.github/tree/master/workflow-templates

### CODE REVIEW

File: .github/workflows/lint-info-xml.yml [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+  xml-linters:
+    runs-on: ubuntu-latest
+
+    name: info.xml lint
+    steps:
+      - name: Checkout
+        uses: actions/checkout@master
+
+      - name: Download schema
+        run: wget https://raw.githubusercontent.com/nextcloud/appstore/master/nextcloudappstore/api/v1/release/info.xsd
```
Comment (nickvergessen) [inline .github/workflows/lint-info-xml.yml:25]: We need to use:
https://github.com/nextcloud/server/blob/master/resources/app-info-shipped.xsd

File: appinfo/info.xml [hunk 2]
```diff
@@ -11,14 +11,13 @@
...
 		More information is available in the Activity documentation.
-	</description>
+</description>
 
 	<version>2.16.0</version>
 	<licence>agpl</licence>
 	<author>Frank Karlitschek</author>
 	<author>Joas Schilling</author>
 
-	<default_enable />
```
Comment (nickvergessen) [inline appinfo/info.xml:21]: This needs to stay, downloading the correct `.xsd` will fix it

Comment (artonge) [reply appinfo/info.xml]: Thanks for the pointer :+1:

**Review by nickvergessen — APPROVED** [review]

**Pull request merged**